formfyxer/data/simplified_words.yml

```diff
@@ -0,0 +1,254 @@
+# Table originates from https://www.plainlanguage.gov/guidelines/words/use-simple-words-phrases/, https://web.archive.org/web/20230102195508/https://www.plainlanguage.gov/guidelines/words/use-simple-words-phrases/
+a and/or b: a or b or both
+accompany: go with
+accomplish: "[carry out, do]"
+accorded: given
```

Comment:
Are these words also supposed to be organized by the length of the key?

Reply:
I did that where it was important to correctness. If you see any short phrases that are above a longer version that incorporates the short one, feel free to fix. I noticed a lot more of those in the gendered terms list.